src/olympia/amo/fields.py

```diff
@@ -10,6 +11,16 @@
 from olympia.amo.widgets import ColorWidget
 
 
+class PositiveAutoField(models.AutoField):
```

Comment:
Can you add a comment somewhere explaining that it needs `olympia.core.db.base` to work? It's not necessarily obvious when you look at it in isolation.